github/src/main/kotlin/gropius/sync/github/model/IssueDataCache.kt

```diff
@@ -5,21 +5,19 @@ import org.bson.types.ObjectId
 import org.springframework.data.annotation.Id
 import org.springframework.data.mongodb.core.index.Indexed
 import org.springframework.data.mongodb.core.mapping.Document
+import java.net.URI
 
 /**
  * Cache for unprocessed issues
+ * @param url API URL of IMS of the repo
```

Comment:
consistent documentation in same class

Reply:
3166ce9ca0b4b2e3cb93224cfc1e38ad88d34a40